### ISSUE TITLE
Feature Request: "disconnect" and "abort" command line actions

### ISSUE THREAD

**Issue by guitarpicva** [la5nta/pat#172]
Propose these two actions to further abstract the two processes from the end user across all OS's, and (selfishly) to allow shell scraping maniacs like me to create UI controls to accomplish the same.

It would seem to me to add a "completeness" to the pat command set as well.  All actions "in pat" are then text commands.

**Comment by wb7ubc** [comment]
Yes, please add the "Stop" and "Abort" CLI commands or radio buttons to PAT so that marginal, futile Winlink connections can be terminated early rather than having to wait for the connection to die a slow, retry-out death.

**Comment by xylo04** [comment]
Some work was done on this for the GUI in #24. Now this just needs CLI support.

**Comment by martinhpedersen** [comment]
I think we're all good now that `SIGINT` (ctrl+c) is coupled to the same disconnect/abort feature we have in the web gui. Closing 😊
